fs.go

```diff
@@ -29,13 +30,19 @@ func (ca *Cagent) FSWatcher() *FSWatcher {
 
 	ca.fsWatcher = &FSWatcher{
 		AllowedTypes:      map[string]struct{}{},
+		ExcludePath:       make(map[string]struct{}),
 		ExcludedPathCache: map[string]bool{},
 		cagent:            ca,
 	}
+
 	for _, t := range ca.Config.FSTypeInclude {
 		ca.fsWatcher.AllowedTypes[strings.ToLower(t)] = struct{}{}
 	}
 
+	for _, t := range ca.Config.FSPathExclude {
+		ca.fsWatcher.ExcludePath[strings.ToLower(t)] = struct{}{}
```

Comment:
I think path should not be lowercased here. Case matters on Linux

Reply:
done